packages/dicomImageLoader/src/imageLoader/wadouri/dataset-from-partial-content.ts

```diff
@@ -52,7 +57,7 @@ function parsePartialByteArray(byteArray) {
     // metadata header and regular datasets, so transfer syntax and
     // other metadata headers aren't included.
     pixelDataSet = dicomParser.parseDicom(byteArray);
-  } catch (err) {
+  } catch (err: any) {
```

Comment:
This is probably invalid handling - it expects the only reason to fail is a partial dataset.  That being said, it isn't part of this change, so it should be handled later.